celesta-sql/src/main/java/ru/curs/celesta/score/discovery/ScoreByScoreResourceDiscovery.java

```diff
@@ -89,4 +90,19 @@ String getGrainName(String grainPath) {
         return result;
     }
 
+    Namespace getGrainNamespace(String grainPath) {
+
+        String value = grainPath.toLowerCase();
+
```

Comment:
The same concern about `null` Namespace.

Reply:
2. Done. You're right. We need to keep up with the time and use all that surgar features of the Java langauage. I justify myself only for the reason of saving my time since I was tired (lazy ?) to perform all that mental streams gymnastics.

Reply:
Well generally I don't like using streams for streams' sake. 

However,  in this particular case, streams give us an opportunity to write two idiomatically similar `getGrainNamespace`  methods in a way that their code also looks similar, so why not using streams?